install/install-base.php

```diff
@@ -131,4 +134,41 @@ public function new_site( $new_site ) {
 		$this->_activate();
 		restore_current_blog();
 	}
+
+	/**
+	 * Returns the plugin's options.
+	 *
+	 * @since 3.7
+	 *
+	 * @return Options
+	 */
+	protected function get_options(): Options {
+		if ( isset( $GLOBALS['polylang'], $GLOBALS['polylang']->model, $GLOBALS['polylang']->model->options ) ) {
+			return $GLOBALS['polylang']->model->options;
+		}
+
+		add_action( 'pll_init_options_for_blog', array( Options_Registry::class, 'register_options' ), 80 ); // Custom priority.
```

Comment:
```suggestion
		add_action( 'pll_init_options_for_blog', array( Options_Registry::class, 'register_options' ), 80 ); // Custom priority to avoid conflict with the hook added in `Polylang` class.
```